resources/qml/SidebarTooltip.qml

```diff
@@ -36,7 +36,7 @@ UM.PointingRectangle {
             }
         }
         base.opacity = 1;
-        target = Qt.point(40 , position.y + Math.round(UM.Theme.getSize("tooltip_arrow_margins").height / 2))
+        target = Qt.point(position.x + 1, position.y + Math.round(UM.Theme.getSize("tooltip_arrow_margins").height / 2))
```

Comment:
I'm not sure. Do we have to round the coordinates here? We're dividing something by 2.

Reply:
The division is already rounded.

Reply:
3 (rounded) / 2 = 1.5 (not rounded)